### PR TITLE
SampleGen: namespace and imports in C#

### PR DESCRIPTION
**Pull request by hzyi-google**

### CODE REVIEW

**Comment by hzyi-google** [comment]
PTAL @googleapis/samplegen

File: src/main/java/com/google/api/codegen/transformer/csharp/CSharpSampleImportTransformer.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
+/* Copyright 2018 Google LLC
```
Comment (vchudnov-g) [inline src/main/java/com/google/api/codegen/transformer/csharp/CSharpSampleImportTransformer.java]: 2019

File: src/main/java/com/google/api/codegen/transformer/csharp/CSharpSampleImportTransformer.java [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+      typeTable.getAndSaveNicknameFor(node.getType());
+      switch (node.getLineType()) {
+        case ReadFileInitLine:
+          typeTable.saveNicknameFor("System.IO.File");
+          break;
+        case SimpleInitLine:
+        case StructureInitLine:
+        case ListInitLine:
+        case MapInitLine:
+          break; // fall through
```
Comment (vchudnov-g) [inline src/main/java/com/google/api/codegen/transformer/csharp/CSharpSampleImportTransformer.java]: `break; // fall through` is confusing

You probably want each of the lines above to have the comment:
```
case SimpleInitLine:  // fall-through
...
caseMapInitLine:
  break;
```

**Comment by chrisdunelm** [comment]
Sorry, I managed to overlook this PR. I'll review as soon as I can.

File: src/test/java/com/google/api/codegen/gapic/testdata/csharp/csharp_library.baseline [hunk 3]
```diff
@@ -17,27 +17,33 @@
...
-    {
-        LibraryServiceClient libraryServiceClient = await LibraryServiceClient.CreateAsync();
-        ShelfName name = new ShelfName("[SHELF_ID]");
-        await libraryServiceClient.DeleteShelfAsync(name);
-        // FIXME: inspect the results
-    }
-    // [END sample_core]
+    // [START sample]
+    using Google.Example.Library;
+    using apis = Google.Example.Library.V1;
```
Comment (beccasaurus) [inline src/test/java/com/google/api/codegen/gapic/testdata/csharp/csharp_library.baseline]: Where did you copy this from?

[Should be](https://github.com/GoogleCloudPlatform/dotnet-docs-samples/blob/8665b94b10f8ad99b09004a180f21209688a05f9/speech/api/Recognize/Recognize.cs#L19):
```cs
using Google.Example.Library.V1;
```

File: src/main/java/com/google/api/codegen/transformer/csharp/CSharpSampleImportTransformer.java (hunk 2 above)
Comment (hzyi-google) [reply src/main/java/com/google/api/codegen/transformer/csharp/CSharpSampleImportTransformer.java]: Actually, found a better way using `Stream`

**Comment by hzyi-google** [comment]
PTAL @beccasaurus @vchudnov-g

**Pull request merged**